tsconfig.json

```diff
@@ -3,22 +3,18 @@
   "compilerOptions": {
     "lib": ["dom", "es2016"],
     "module": "esnext",
-
     /* Output options */
     "noEmit": true,
-
+    "allowUmdGlobalAccess": true,
```

Comment:
Whoops, don't think this was needed. (was part of me trying to get global typescripts to be seen by eslint)